src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -496,6 +503,19 @@ public YesNo getRegulation35Selection() {
         }
     }
 
+    @JsonIgnore
+    public YesNo getSchedule9Paragraph4Selection() {
+        if (sscsUcCaseData != null) {
+            if ("No".equalsIgnoreCase(sscsUcCaseData.getUcWriteFinalDecisionSchedule7ActivitiesApply())) {
+                return doesSchedule9Paragraph4Apply;
+            } else {
+                return null;
+            }
+        } else {
+            return null;
+        }
```

Comment:
this block ``` } else {
                return null;
            }
        } else {
            return null;
        }``` could be simplified by one return null at the bottom of the method. i.e. not having any else statements.